compiled_starters/kotlin/.gitignore

```diff
@@ -0,0 +1 @@
+kotlin_redis.jar
```

Comment:
@urielsalis do you think it'd be okay to use the standard .gitignore that GitHub uses? https://github.com/github/gitignore/blob/main/Java.gitignore

I can make the change, just wanted to confirm if you think this is okay. 

I saw that this and bittorrent had a different .gitignore for example, so thought it'd be worth standardizing

Reply:
For both Java and Kotlin the gitignore would look the same

The one for Java is a good starting point, but probably needs to be combined with the maven one (https://github.com/github/gitignore/blob/main/Maven.gitignore) and the Intellij one https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore (first part can be simplified to just .idea/)